pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/github_scraper.go

```diff
@@ -114,7 +375,7 @@ func (ghs *githubScraper) scrape(ctx context.Context) (pmetric.Metrics, error) {
 
 	// TODO: setting this here for access from the proceeding for statement
 	// gathering repo data
-	var searchRepos []getRepoDataBySearchSearchSearchResultItemConnectionEdgesSearchResultItemEdge
+	var searchRepos []SearchNode
```

Comment:
`SearchNode` looks so much better. I thought we might've already had search node defined in genqlient. There's not two values with that name is there?

Reply:
Nope, just the one luckily